### PR TITLE
adds a more mobile friendly user interface to chat.jsp

### PR DESCRIPTION
**Pull request by MiscLG**
I mainly focused on adding the user interface to chat.jsp because I'm more familiar with it, and I think it works well for now at least. I'm pretty sure some of this will cause merge conflicts once we merge Lizzy's pull request so I'll edit this request once the other one is merged. Also I would like to change the nav bar to say Chats instead of Conversations for styling's sake.

### CODE REVIEW

**Comment by MiscLG** [comment]
Some previews of what the code does!
mobile
mobiletype

**Comment by ElizabethRiffle** [comment]
The preview looks good to me! I am also a fan of changing "conversations" to "chat."

**Comment by tbiscuit** [comment]
Looks like it's mostly doing what you've shown above but when I run it locally the CodeU Chat App & similar text doesn't autocenter the way you have it in the screenshot. Not sure if it's something unique to my local environment or not.

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -18,24 +18,28 @@ limitations under the License.
...
+        <a href="/">Logout</a>
+        <a href="/about.jsp">About</a>
         <% } else{ %>
-            <a href="/login">Login</a>
-            <a href="/register">Register</a>
+        <a href="/login">Login</a>
+        <a href="/register">Register</a>
+        <a href="/about.jsp">About</a>
         <% } %>
-    </nav>
```
Comment (ElizabethRiffle) [inline src/main/webapp/index.jsp:38]: Some of the indenting on this nav bar is a little funky and makes it harder to read so I would maybe fix this before merging!

**Pull request merged**